src/account/PluginManager.sol

```diff
@@ -27,17 +27,20 @@ import {
     PluginManifest
 } from "../interfaces/IPlugin.sol";
 
-abstract contract PluginManagerInternals is IPluginManager {
+contract PluginManager {
```

Comment:
What're the tradeoffs between this approach vs library external function?

Reply:
Using a library with external functions is a bit less straightforward to deploy through foundry. This pattern of manually `delegatecall`ing a custom contract also lets us inspect calls between the account and the plugin manager implementation better, because we know the abi-encoding format. Libraries would use the solidity-internal custom encoding for libraries: https://docs.soliditylang.org/en/v0.8.19/contracts.html#function-signatures-and-selectors-in-libraries

Also, as a note for the future: this change is intended to make it easier to work with the reference implementation as of the current version: Spec Update 6, aka v0.6.0. We hope to remove this workaround as the standard gets simplified in the future version v0.7.0 and have a single-contract reference implementation, even without optimizations.